pt-BR/active_record_migrations.md

```diff
@@ -883,36 +883,23 @@ Como alternativa, você deveria escrever uma nova *migration* que execute as mud
 
 O método `revert` pode ajudar ao escrever uma nova *migration* para desfazer *migrations* anteriores no todo ou em partes (veja [Revertendo *Migrations* Anteriores](#revertendo-migrations-anteriores) acima).
 
-Schema Dumping and You
```

Comment:
Nas pesquisas que fiz `dump` sempre era traduzido como coisas relacionadas a "lixo", como "aterro", "descarte", "despejo", "entulho".  
Achei melhor manter o termo original por isso, mas caso tenham alguma sugestão fiquem à vontade. xD

Reply:
> Nas pesquisas que fiz `dump` sempre era traduzido como coisas relacionadas a "lixo", como "aterro", "descarte", "despejo", "entulho".
> Achei melhor manter o termo original por isso, mas caso tenham alguma sugestão fiquem à vontade. xD

Também acho legal manter o termo original. Talvez podemos também colocar um link externo explicando mais o termo, assim fica bem didático. O que acham @campuscode/guia-rails-team @L-Ghost ?

Reply:
Concordo com as sugestões: podemos deixar o termo como `*Schema Dumping*`, em itálico, e colocar um link extra com mais informações sobre o termo. :)

Reply:
A página mais relevante e mais neutra que julguei que explica melhor (em português) o que seria um "dump" foi [esse artigo na Wikipedia mesmo](https://pt.wikipedia.org/wiki/Dump_de_banco_de_dados). Linkei ele no primeiro parágrafo da seção.

Reply:
> Wikip

Excelete!